devices/stm32f3x8.yaml

```diff
@@ -1,5 +1,125 @@
 _svd: ../svd/stm32f3x8.svd
 
+RCC:
+  CFGR:
+    _delete:
+      - ADCPRE
+      - SDPRE
+    _add:
+      I2SSRC:
+        description: I2S external clock source selection
+        bitOffset: 23
+        bitWidth: 3
```

Comment:
I think this is only 1 bit wide?

Reply:
Good catch ! Once again !
Fixed.
I really want to thank you for your throughout reviews. This kind of review must be boring, but it proves really useful to push the quality of those crates one step above.